node/core/dispute-coordinator/src/db/v1.rs

```diff
@@ -79,12 +99,50 @@ impl Backend for DbBackend {
 		for op in ops {
 			match op {
 				BackendWriteOp::WriteEarliestSession(session) => {
+					// Cleanup old votes in db:
+					let watermark =
```

Comment:
Nit: I would move this into a separate fn like `periodic_vote_cleanup` to make the code more readable.

Reply:
I was on the edge ;-) Will do!